src/main/java/org/synyx/urlaubsverwaltung/sicknote/web/SickNoteForm.java

```diff
@@ -0,0 +1,160 @@
+package org.synyx.urlaubsverwaltung.sicknote.web;
+
+import org.synyx.urlaubsverwaltung.period.DayLength;
+import org.synyx.urlaubsverwaltung.person.Person;
+import org.synyx.urlaubsverwaltung.sicknote.SickNote;
+import org.synyx.urlaubsverwaltung.sicknote.SickNoteAction;
+import org.synyx.urlaubsverwaltung.sicknote.SickNoteComment;
+import org.synyx.urlaubsverwaltung.sicknote.SickNoteType;
+
+import java.time.LocalDate;
+
+/**
+ * View class representing an sick note.
+ */
+public class SickNoteForm {
+
+    private Integer id;
+
+    private Person person;
+
+    private SickNoteType sickNoteType;
+
+    private LocalDate startDate;
+
+    private LocalDate endDate;
+
+    private DayLength dayLength;
+
+    private LocalDate aubStartDate;
+
+    private LocalDate aubEndDate;
+
+    private String comment;
```

Comment:
wie machen wir das denn in der urlaubsverwaltung mit immutability?

Reply:
mir ist bisher keine konkrete Überlegung zu immutability und Urlaubsverwaltung bekannt / bewusst